web/src/stream_settings_data.ts

```diff
@@ -36,7 +39,8 @@ export function get_sub_for_settings(sub: StreamSubscription): SettingsSubscript
             new Date(sub.date_created * 1000),
             "dayofyear_year",
         ),
-        creator: stream_data.maybe_get_creator_details(sub.creator_id),
+        creator,
+        is_active: creator ? people.is_person_active(creator.user_id) : false,
```

Comment:
This can be confused with being used for whether the stream is active or not. Maybe `is_active` field can be added to `creator` object.